views/index.erb

```diff
@@ -0,0 +1,3 @@
+<% @apartments.each do |apartment| %>
+<%= "<li><a href='/apartment/#{apartment.id}'>#{apartment.address}</a></li>" %>
```

Comment:
Personally, I find writing quotes inside quotes to be really annoying, because I always forget whether I'm supposed to use single quotes or double quotes. You could also write this as:

```
<li><a href="/apartment/<%= apartment.id %>"><%= apartment.address %></a></li>
```